command/src/main/java/com/jagrosh/jdautilities/command/Command.java

```diff
@@ -53,13 +53,13 @@
  * }</code></pre>
  * 
  * Execution is with the provision of a MessageReceivedEvent-CommandClient wrapper called a
- * {@link com.jagrosh.jdautilities.command.CommandEvent CommandEvent} and is performed in two steps:
+ * {@link CommandEvent CommandEvent} and is performed in two steps:
  * <ul>
- *     <li>{@link com.jagrosh.jdautilities.command.Command#run(CommandEvent) run} - The command runs
+ *     <li>{@link Command#run(CommandEvent) run} - The command runs
  *     through a series of conditionals, automatically terminating the command instance if one is not met, 
  *     and possibly providing an error response.</li>
  *     
- *     <li>{@link com.jagrosh.jdautilities.command.Command#execute(CommandEvent) execute} - The command,
+ *     <li>{@link Command#execute(CommandEvent) execute} - The command,
```

Comment:
See earlier comments on JDA documentation standards

Reply:
Although Github doesn't show it this has been fixed.